README.md

```diff
@@ -114,7 +114,7 @@ MSI installers are available for download on the [releases page][].
 Install and upgrade:
 
 1. Download the `.deb` file from the [releases page][]
-2. `sudo apt install ./gh_*_linux_amd64.deb` install the downloaded file
+2. `sudo dpkg -i ./gh_*_linux_amd64.deb` install the downloaded file
```

Comment:
Hi, could you explain what exactly is fixed by this change? Did `apt install` not work for you?

Please see https://github.com/cli/cli/pull/785

Reply:
Yes, apt install refuses to install Debian files. FWIW I am on Ubuntu 18.10. I think given that old versions of apt don't support Debian install, and dpkg -i works for all the versions, I think it's prudent to change the instructions back to the dpkg version.

Reply:
Looking into the related issues, I see that the `./` is important.

Reply:
Feel free to close this PR, but I think that it's worth switching to instructions that are less likely for people to make mistakes with and are compatible with more distros.

Reply:
> I think that it's worth switching to instructions that are less likely for people to make mistakes with

For sure! We are working on offering our `.deb` and `.rpm` packages in a different, more user-friendly way in the near future, so we'll update these installation instructions accordingly.